.coveragerc

```diff
@@ -12,4 +12,7 @@ omit =
     #src/RestrictedPython/tests/*.py
 
 [report]
-precision = 3
+precision = 2
+
+[html]
+directory = reports/coverage
```

Comment:
What is the reason behind this change? I do not believe that we need flake8 reports as we require zero flake8 failures.